packages/cozy-sharing/src/components/EditableSharingModal.jsx

```diff
@@ -58,8 +59,12 @@ const EditableSharingModal = ({
               link={getSharingLink(document.id)}
               permissions={getDocumentPermissions(document.id)}
               isOwner={isOwner(document.id)}
-              hasSharedParent={hasSharedParent(documentPath)}
-              hasSharedChild={hasSharedChild(documentPath)}
+              hasSharedParent={
+                documentPath ? hasSharedParent(documentPath) : false
```

Comment:
I think `documentPath && hasSharedParent(documentPath)` would work as well and may be easier to read?